packages/data-explorer-ui/src/components/common/Banner/banner.tsx

```diff
@@ -0,0 +1,10 @@
+import { Box } from "@mui/material";
+import React, { ReactNode } from "react";
+
+export interface BannerProps {
+  children: ReactNode;
+}
+
+export const Banner = ({ children, ...props }: BannerProps): JSX.Element => {
```

Comment:
Let's remove the prop spreading on this component - we can add it later if we need it. Thank you!

Reply:
The need for this was removed. I used `styled.div` here instead. Was much clean and simpler. Thanks!

Reply:
Hi @imadx, I think we should keep the Banner component as per the ticket [Part A](https://github.com/clevercanary/data-explorer/issues/89) - we may extend the use of the component - add props in the future etc etc. Can you please undo this last change?